dask/base.py

```diff
@@ -982,8 +982,7 @@ def normalize_array(x):
         if hasattr(x, "mode") and getattr(x, "filename", None):
             if hasattr(x.base, "ctypes"):
                 offset = (
-                    x.ctypes.get_as_parameter().value
-                    - x.base.ctypes.get_as_parameter().value
+                    x.ctypes._as_parameter_.value - x.base.ctypes._as_parameter_.value
```

Comment:
Initially I was concerned to see us using a private `_as_parameter_` attribute, however after looking a little closer this is what [NumPy recommends](https://github.com/numpy/numpy/blob/5c19c2350a9d839db835540e52697ea79c52ce47/numpy/core/_internal.py#L383-L392) so this seems okay. Just posting a quick comment here in case other had a similar thought